iphone/Classes/TiUITableView.m

```diff
@@ -1448,6 +1450,52 @@ - (CGFloat)contentHeightForWidth:(CGFloat)suggestedWidth
   return height;
 }
 
+- (void)updateSearchControllerFrames
+{
+  if (![searchController isActive]) {
+    return;
+  }
+
+  CGPoint convertedOrigin = [self.superview convertPoint:self.frame.origin toView:searchControllerPresenter.view];
+  CGFloat topMargin = convertedOrigin.y;
+
+#if IS_XCODE_9
+  if ([TiUtils isIOS11OrGreater]) {
+    topMargin += self.safeAreaInsets.top;
+    tableview.frame = CGRectMake(0, self.safeAreaInsets.top, tableview.frame.size.width, tableview.frame.size.height - self.safeAreaInsets.top);
```

Comment:
Should that always be done? I thought only when having the `extendSafeArea` property set?